plans/sep8.md

```diff
@@ -91,6 +96,8 @@ interface ActionRequired extends ApprovalResponse {
   status: "action_required";
   message: string;
   action_url: string;
+  action_method?: string;
+  action_fields?: string[];
 }
```

Comment:
➕ I imagine you probably need to add types for the new POST responses detailed [here](http://stellar.org/protocol/sep-8#action-url).

Reply:
Question: how do you have a dynamic type based on what's returned in `action_fields`? Or do we simply include all the possible fields in SEP-11?

Reply:
I don't think I understand the question completely, so I might need you to elaborate, but if you're asking what types will be in action_fields, it's an array of strings. The contents are field names for fields in SEP-9, not SEP-11.

Reply:
Ah nvm. I realized you were referring to the POST response, not the request.

> The contents are field names for fields in SEP-9, not SEP-11.

My bad. SEP-11, not SEP-9.

My original question was about how we can construct a request type that is based on what's in the array of strings.